src/components/Matches/FileExporter/index.tsx

```diff
@@ -74,21 +76,10 @@ const FileExporter = (
           search.extra_fields ? `,${search.extra_fields}` : ''
         }`,
       }}
-      endpoint={(endpoint[primary] && endpoint[primary][secondary]) || primary}
+      endpoint={(endpoint?.[primary]?.[secondary] as string) || primary}
       minWidth={minWidth}
```

Comment:
Passing `label` to `File`:

```suggestion
      minWidth={minWidth}
      label={label}
```